### ISSUE TITLE
[Bug]: Livewire namespace

### ISSUE THREAD

**Issue by hamzaelmaghari** [laravel-idea/plugin#620]
### Bug description

Hi,
Livewire components namespace doesn't work properly.

# The field contains elements:

<img width="420" alt="Screenshot 2022-08-23 at 23 58 44" src="https://user-images.githubusercontent.com/84163115/186280643-42c2bb27-e500-4a34-a596-fb061ddf56e9.png">

# The config contains elements

<img width="656" alt="Screenshot 2022-08-24 at 00 01 16" src="https://user-images.githubusercontent.com/84163115/186280789-e4d4081a-7577-4d8b-8b26-84f34b2848a9.png">

# The namespace Folders
<img width="315" alt="Screenshot 2022-08-23 at 23 59 02" src="https://user-images.githubusercontent.com/84163115/186280841-b47ad7ac-99b5-4261-9f7a-accfb622f5e1.png">

### Plugin version

5.7.3.221

### Operating system

MacOS

**Comment by adelf** [comment]
Hello.

You have class_namespace - "App\\Http\\Elements", but I don't see this namespace in your project. There is 'app/Http/Livewire'.

**Comment by hamzaelmaghari** [comment]
> 

Yes, that's the problem when creating components with ( Laravel Idea ).
It seems like the plugin ignores `Components tag namespace field`
